loan-payoff/REPWRITERSPECS/BANNO.LOAN.PAYOFF.FEES.POW

```diff
@@ -0,0 +1,195 @@
+[*
+**  PowerOn Name:       BANNO.LOAN.PAYOFF.FEES.POW
+**
+**  Copyright 2022 Jack Henry and Associates
+**
+**  This Banno subroutine PowerOn is called by the main BANNO.LOAN.PAYOFF.V1.POW PowerOn -- if configured
+**  in the CFG letterfile. It can be modified to return custom fee amounts and descriptions to the UX for display.
```

Comment:
Probably better to specify the entire PO filename here instead of just 'CFG letterfile' to avoid any ambiguities.

Reply:
Done.